### DISCUSSION TITLE
[v4] Using theme() inside @media with Sveltekit

### DISCUSSION THREAD

**Discussion by minht11** [tailwindlabs/tailwindcss#15129] [Help] [answered]
I am trying out new v4 version and trying to replace `@screen sm` with `@media (width >= theme(--breakpoint-sm))`
According to docs it should be possible? Am I doing something wrong? Styles are inside sveltekit.
<img width="775" alt="image" src="https://github.com/user-attachments/assets/bea92921-4d8d-41c8-8595-8cf6306fcc02">

**Comment by wongjn** [comment]
No, seems like it should work: https://play.tailwindcss.com/gcMeZa6KXZ

**Comment by adamwathan** [comment, marked as answer]
@minht11 Are you doing this in a `<style>` block in a Svelte component? If so make sure you are importing your theme so that the variables are available.

I didn't mention this explicitly in the docs (I should update them) but it's the same principle as using `@apply` in Svelte components:

https://tailwindcss.com/docs/v4-beta#using-apply-in-vue-svelte

Every `<style>` block is processed by SvelteKit in total isolation, so defining `@theme` stuff in one CSS file doesn't make them available elsewhere, so you have to explicitly import anything you need access to for things to work.

**Reply by minht11** [reply]
Upgraded to 4.0-beta.5 and with
```
@import '../../app.css' reference;
```
it works correctly thanks.
